scripts/cita

```diff
@@ -2,17 +2,32 @@
 # -*- tab-width:4;indent-tabs-mode:nil -*-
 # ex: ts=4 sw=4 et
 
+# Commands Paths
+CITA_BIN="$(cd $(dirname "$0") && pwd -P)"
+CITA_SCRIPTS="$(dirname $CITA_BIN)/scripts"
+if [ "$1" != "bebop" ]; then
+    if [ -e $CITA_BIN/cita-env ];then
+        $CITA_BIN/cita-env bin/cita bebop $@
```

Comment:
The `bebop` keyword is my selfish addition, it's from a famous cartoon in 1998, [cowboy bebop][1], galaxy adventure, Rock and Jazz, sounds like [this][2].

If I need to modify it to another expression, it's okay.

[1]: https://en.wikipedia.org/wiki/Cowboy_Bebop
[2]: https://music.163.com/#/song?id=592687

Reply:
so `bebop` is used for using docker, right?

I'd like to use your choice on this pr for thanks of your contribution.

Btw, Maybe will change it to a more easy-understanding word in the future, i'm not suer about this.

Reply:
> so bebop is used for using docker, right?
Yep, `bebop` is only for commands inside docker and users using cita through docker may never reach it.

> Btw, Maybe will change it to a more easy-understanding word in the future, i'm not suer about this.
It's fine, thank you.